xml-parsing/src/test/resources/checks/SimpleXPathBasedCheck/xPathFailure.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!DOCTYPE foo [<!ELEMENT foo ANY >
+        <!ENTITY xxe SYSTEM "file:///dev/random" >]>
+
+<foo>&xxe;</foo>
```

Comment:
it  will be nice to add comment here about how this file fails and that we don't know why.